### PR TITLE
feat(insured-bridge): Add additional ChainId props

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

Previously, we had hardcoded chainIDs into the OVM/AVM bridge deposit boxes. This does not work given we want to deploy the same contract on the testnet/mainnet versions of a given chain. This PR adds the ability to define the chainID on these deposit boxes.

**Summary**

Adds chainID prop. Note no validation happens on the chainIDs passed in as the Admin is expected to configure this correctly at deployment time.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [X]  Untested

### CODE REVIEW

File: packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // from the respective chains and return it to client implementors.
 
 import { getAbi } from "@uma/contracts-node";
-import type { OVMBridgeDepositBoxWeb3 } from "@uma/contracts-node";
+import type { BridgeDepositBoxWeb3 } from "@uma/contracts-node";
```
Comment (chrismaree) [inline packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts:5]: unrelated change - made for consistancy. The client does not need `OVM` box, just the normal `BridgeDepositBox`

**Review by nicholaspai — APPROVED** [review]
LGTM! much needed cleanup
